obelix/idefix/devAuth/gclient.go

```diff
@@ -0,0 +1,56 @@
+package devAuth
+
+import (
+	"context"
+	"fmt"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"net/http"
+	"os"
+)
+
+var (
+	CLIENT_ID     = ""
+	CLIENT_SECRET = ""
+	REDIRECT_URL  = ""
+)
+
+func init() {
+	CLIENT_ID = os.Getenv("GOOGLE_CLIENT_ID")
+	CLIENT_SECRET = os.Getenv("GOOGLE_CLIENT_SECRET")
+	REDIRECT_URL = os.Getenv("GOOGLE_REDIRECT_URL")
+}
+
+type Client struct {
+	ctx context.Context
+	Clt *http.Client
+}
+
+func (c *Client) GetCalendarList() (*http.Response, error) {
+	get, err := c.Clt.Get("https://www.googleapis.com/calendar/v3/users/me/calendarList")
+	if err != nil {
+		return nil, fmt.Errorf("failed to get calendar list: %v", err)
+	}
+	return get, nil
+}
```

Comment:
Please use the official golang calendar package available [here](https://developers.google.com/calendar/api/quickstart/go).

Reply:
dead code

Reply:
6572d568177a397940d5454fb1d43e8c72a00a97